README.md

```diff
@@ -1,2 +1,16 @@
 # VanRoomies
 A match-making housing app
+
+## BACKEND 
+### Development
+#### Set up development key and cert
+**Linux**
+Change directory into backend.
+```bash
+$ cd backend/
+```
+Generate a certificate and key.
+```bash
+$ openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -sha256 -days 365
```

Comment:
This will require the IP of the VM so it is allowed to be trusted by the Frontend, Ill try find the relevant post for this on how to add the IP of the VM as alttext to the self signed certificate

Reply:
So I think [this](https://medium.com/@antelle/how-to-generate-a-self-signed-ssl-certificate-for-an-ip-address-f0dd8dddf754) is what I was referring to. At the very end there's a handy script we can invoke from curl. Ive read through the script and it is safe to use - plainly generating the cert and key and adding the IP we provide as a verified. Setting this aside, the https server standalone works fine. Able to run it it and I can make requests through Postman to the `https://localhost:3000` and some endpoint and it functions as expected.